docs/en/week09/09-3.md

```diff
@@ -72,7 +72,7 @@ $$
 As the discriminator becomes more confident, $D(\vect{x})$ gets closer to $1$ and $D(\vect{\hat{x}})$ gets closer to $0$. This confidence moves the outputs of the cost network into flatter regions where the gradients become more saturated. These flatter regions provide small, vanishing gradients that hinder the generator network's training. Thus, when training a GAN, you want to make sure that the cost gradually increases as you become more confident.
 
 
-### 3. Mode collapse
+### 3. Model collapse
```

Comment:
Nope. This is *mode collapse*. See [mode](https://www.oxfordlearnersdictionaries.com/definition/english/mode?q=mode).

Reply:
I changed it.